sdk/src/main/java/io/radar/sdk/RadarSettings.kt

```diff
@@ -311,15 +313,41 @@ internal object RadarSettings {
         return RadarFeatureSettings.fromJson(JSONObject(optionsJson))
     }
 
+    fun setSdkConfiguration(context: Context, configuration: RadarSdkConfiguration?) {
+        Radar.logger.d("set SDK Configuration | sdkConfiguration = $configuration")
+        if (configuration != null) {
+            getSharedPreferences(context).edit { putInt(KEY_LOG_LEVEL, configuration.logLevel.value) }
```

Comment:
I think we want to store the entire object?

Reply:
in https://github.com/radarlabs/radar-sdk-android/pull/365

Reply:
not needed for logLevel, added in the next PR